.buildkite/scripts/steps/k8s-extended-tests.sh

```diff
@@ -26,7 +26,7 @@ else
 fi
 
 SNAPSHOT=true EXTERNAL=true PACKAGES=docker mage -v package
-TEST_INTEG_CLEAN_ON_EXIT=true INSTANCE_PROVISIONER=kind STACK_PROVISIONER=stateful SNAPSHOT=true mage integration:kubernetesMatrix
+TEST_INTEG_CLEAN_ON_EXIT=true INSTANCE_PROVISIONER=kind STACK_PROVISIONER=stateful SNAPSHOT=true mage integration:testKubernetesMatrix
 TESTS_EXIT_STATUS=$?
```

Comment:
`nit`: as you touch the sh script, do you think we can enable the BK log group?

```suggestion
TESTS_EXIT_STATUS=$?

if [[ $exit_status -ne 0 ]]; then
  echo "^^^ +++"
fi
```

Reply:
@v1v I do that exact thing [here](https://github.com/elastic/elastic-agent/pull/8484/files#diff-59087217e78ee67e56ab89fae2f04a36447bcb4fa7753ad4491f8521901eaadbR106-R109), right? 🙂

Reply:
oh no this is the other k8s-extended-tests.sh, this isn't used anywhere AFAICT 🙂 , but I am not cleaning it up just yet, not in this PR 🙂